apps/nextjs-example/package.json

```diff
@@ -31,8 +31,8 @@
     "@trustwallet/aptos-wallet-adapter": "^0.1.6",
     "@welldone-studio/aptos-wallet-adapter": "^0.1.4",
     "antd": "^5.1.2",
-    "aptos": "^1.19.0",
-    "@aptos-labs/ts-sdk": "^0.0.3",
```

Comment:
this looks pretty old, it already uses 0.0.7 
https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/apps/nextjs-example/package.json#L14

Reply:
Yeah, I was using a really old version, since been fixed